### ISSUE TITLE
Lein run gives null pointer

### ISSUE THREAD

**Issue by jellea** [swlkr/coast#5]
`lein repl` + `(coast)` works, but `lein run` does not.

```> lein run
:reloading (coasttest.components coasttest.views.errors coasttest.controllers.errors-controller coasttest.views.home coasttest.controllers.home-controller coasttest.routes coasttest.core coasttest.core-test)
Server is listening on port 1337
Mon Jan 22 17:15:16 WET 2018 [worker-1] ERROR - GET /
java.lang.NullPointerException
	at org.httpkit.server.HttpHandler.run(RingHandler.java:91)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
```

**Comment by swlkr** [comment]
Oh man, another killer. Well that stinks, give me a while to look into this, or if you feel like tackling this one too... 🥇

**Comment by swlkr** [comment]
Hey I fixed this in 0.6.6 ! Updated the lein template as well. Closed with b259e5a102b77cbaecab4e0b80af0ce8818c4002
